IF/src/main/java/com/github/stefvanschie/inventoryframework/pane/Pane.java

```diff
@@ -370,9 +368,9 @@ public static GuiItem loadItem(@NotNull Object instance, @NotNull Element elemen
                                 if (!innerNode.getNodeName().equals("enchantment"))
                                     continue;
 
-                                Enchantment enchantment = Enchantment.getByKey(NamespacedKey.minecraft(
+                                Enchantment enchantment = Enchantment.getByName(

```

Comment:
I mean, making a different method in which the `getByName` and `getByKey` used depending on the version, so it will not have any conflict or non-existing enchantment exception.